OSCAL_REFERENCE.md

```diff
@@ -19,7 +19,11 @@ For docker-trestle, the `control-implementation` section is maintained and edite
 | `title` | Title of the SSPP document |
 | `last-modified` | Do not edit - automatically generated timestamp |
 | `version` | Version of the SSP document. Revise whenever submitting final doc for approvals |
-| `oscal-version` | Version of OSCAL in use. trestle currently supports `1.0.4` |
+| `oscal-version` | Version of OSCAL in use. trestle currently supports `1.1.2` |
+| `revisions` | Version history of the document. Used to populate `Document Revision History` table in the templates |
+| `roles` | Names of roles referenced within `responsible-parties` lists. |
```

Comment:
suggestion (non-blocking): this is only one with a period
```suggestion
| `roles` | Names of roles referenced within `responsible-parties` lists |
```

Reply:
I _really_ wish I could accept this without removing your approval...